tests/python/test_relational_vcfs.py

```diff
@@ -0,0 +1,90 @@
+import nose
+
+import tempfile
+
+
+
+def test_vcf_to_csv():
+    import vcf, csv
+    from common.relational_vcf import vcf_to_csv
+    vcfreader = vcf.Reader(open('tests/data/snv.vcf'))
+    columns = ['contig', 'position', 'sample:DP', 'info:VAF', 'info:DP', 'something']
+    filename = vcf_to_csv(vcfreader, columns, None,
+                          default_values={'something': 'Something!'})
+
+    with open(filename) as fd:
```

Comment:
I always think of file descriptors as numbers. Do they usually refer to streams in Python?